.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.0
 jobs:
   test:
+    resource_class: large
```

Comment:
Previous class wasn't enough? Or is it about the time it takes?

Reply:
Wasn't enough, I think that probing was too heavy